roles/edpm_multipathd/tasks/install.yml

```diff
@@ -14,37 +14,33 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-- name: Gather user fact
+- name: Gather ansible_local facts
   ansible.builtin.setup:
     gather_subset:
       - "!all"
       - "!min"
-      - "user"
-      - "user_id"
+      - "local"
   when:
-    - ansible_user is undefined
+    - ansible_local is not defined
+
+- name: Clean up legacy containerized multipathd
+  ansible.builtin.include_tasks: cleanup_containers.yml
 
 - name: Run multipathd install tasks with root privileges
   become: true
   block:
-    - name: Gather services facts
-      ansible.builtin.service_facts:
-
-    - name: Stop multipathd service on the host
-      ansible.builtin.systemd:
-        name: "{{ item }}"
-        state: stopped
-        enabled: false
-      when:
-        - ansible_facts.services["multipathd"] is defined
-        - ansible_facts.services["multipathd"]["status"] != "not-found"
-        - ansible_facts.services["multipathd"]["status"] == "enabled"
-      failed_when: false
-      loop:
-        - multipathd.service
-        - multipathd.socket
-      loop_control:
-        index_var: multipath_service_index
+    - name: Install multipathd packages
+      tags:
+        - install
+        - edpm_multipathd
+      ansible.builtin.dnf:
+        name: "{{ edpm_multipathd_packages }}"
```

Comment:
I double checked and this package is already in the bootc image :+1: 
https://github.com/openstack-k8s-operators/edpm-image-builder/blob/ee5219d7df4772586105649e03d1f545c9e4d653/bootc/Containerfile#L35